README.md

```diff
@@ -45,15 +45,25 @@ npm install @tokens-studio/sd-transforms
 
 > Note: this library is available both in CJS and ESM
 
+## create a file to run custom transformers 
+
 ```js
 const { registerTransforms } = require('@tokens-studio/sd-transforms');
-const StyleDictionary = require('style-dictionary');
+// import style dictionary and the config for it.
+const StyleDictionary = require('style-dictionary').extend(
+  './style-dictionary-config.json',
+);
 
 // will register them on StyleDictionary object
 // that is installed as a dependency of this package.
 registerTransforms(StyleDictionary);
 ```
 
+## Now to run your transformers use 
+```
+node name-of-the-file.js
+```
+
```

Comment:
This is quite confusing. What is name-of-the-file.js? If it's the snippet right above this, that won't do anything? It creates the SD instance and runs the transforms, but then what?